src/chrome/content/rules/Hendrik.luup.info.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="Hendrik Luup">
+  <target host="hendrik.luup.info" />
+
+  <rule from="^http://hendrik\.luup\.info/" to="https://hendrik.luup.info/" />
```

Comment:
You should update this rule to use the new rewrite format

rule from="^http:" to="https:"/>